run-qa-checks

```diff
@@ -5,3 +5,15 @@ set -e
 openwisp-qa-check \
         --migration-path "./openwisp_radius/migrations" \
         --migration-module openwisp_radius
+
```

Comment:
I think in this script we also have to perform the following checks:

- https://github.com/openwisp/django-freeradius/blob/master/.travis.yml#L36
- https://github.com/openwisp/django-freeradius/blob/master/.travis.yml#L58